metro.config.js

```
@@ -9,15 +9,50 @@ const { withNativeWind } = require("nativewind/metro");
+const defaultConfig = getDefaultConfig(__dirname);
+
+const config = {
+  resolver: {
     extraNodeModules: {
+      assert: require.resolve('empty-module'), // assert can be polyfilled here if needed
+      http: require.resolve('empty-module'), // stream-http can be polyfilled here if needed
+      https: require.resolve('empty-module'), // https-browserify can be polyfilled here if needed
+      os: require.resolve('empty-module'), // os-browserify can be polyfilled here if needed
+      url: require.resolve('empty-module'), // url can be polyfilled here if needed
+      zlib: require.resolve('empty-module'), // browserify-zlib can be polyfilled here if needed
+      path: require.resolve('empty-module'),
```

Biome (1.9.4)

[error] This property value named path is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named path


(lint/suspicious/noDuplicateObjectKeys)